itests/testdata/transfer_tx_testdata.go

```diff
@@ -49,38 +71,387 @@ type CommonTransferData struct {
 	NFT   TransferTestData[TransferExpectedValuesPositive]
 }
 
-func GetCommonTransferData(suite *f.BaseSuite, assetId crypto.Digest) CommonTransferData {
-	commonAmount := utl.GetAssetBalanceGo(suite, utl.GetAccount(suite, 2).Address, assetId) / 4
+func GetCommonTransferData(suite *f.BaseSuite, assetId *crypto.Digest, accountNumbers ...int) CommonTransferData {
```

Comment:
Are you sure that `accountNumbers` is necessary here? I don't see any usages of `GetCommonTransferData` where `accountNumbers` argument is not empty.

Reply:
Yes, I'm sure, this method with  such method signature will be used in more complicated test cases later.